CHANGES.rst

```diff
@@ -11,6 +11,8 @@ Major changes
   - SciPy (>= 1.2)
   - scikit-learn (>= 0.20.0)
 
+* Add SuperVectorizer v1
```

Comment:
Desribe here what the SuperVectorizer does

Reply:
Addressed in 6aa83cf